TLM/TLM/Manager/Impl/ExtUITabStrip.cs

```diff
@@ -0,0 +1,154 @@
+// tabbing code is borrowed from RushHour mod and Advanced toolbar mod
+// https://github.com/PropaneDragon/RushHour/blob/release/RushHour/Options/OptionHandler.cs
+// https://github.com/CWMlolzlz/CS-AdvancedToolbar/blob/master/Source/ExpandableToolbar.cs
+
+namespace TrafficManager.Manager.Impl {
+    using UnityEngine;
+    using ICities;
+    using ColossalFramework.UI;
+
+    public sealed class ExtUITabstrip : UITabstrip {
+        private UIScrollbar CreateVerticalScrollbar(UIPanel panel, UIScrollablePanel scrollablePanel) {
```

Comment:
If it's possible move scrollbar more to the right edge of options window

Reply:
Its as right as it can be.